util-scripts/build.sh

```diff
@@ -158,9 +158,9 @@ if [[ $spec_prj == "" ]]; then
   for prj in $projects; do
     to_build="$to_build cardano-sl-$prj"
   done
-  to_build="$to_build cardano-sl"
+  to_build="$to_build cardano-sl cardano-sl-lwallet"
 elif [[ $spec_prj == "sl" ]]; then
-  to_build="cardano-sl"
+  to_build="cardano-sl cardano-sl-lwallet"
```

Comment:
@neongreen are there reasons to do it this way instead of putting it into `projects` variable?